library/lib/components/Sidebar.tsx

```diff
@@ -1,5 +1,8 @@
 import React, { DragEvent } from "react"
-import { dropElementConfig, transformScale } from "@/constants"
+import {
+  dropElementConfig,
+  transformScale,
+} from "@/constants/dropElementConfig"
```

Comment:
for all constants imports I prefer this kind of usage but it is not important
```suggestion
} from "@/constants"
```